ice_setup/ice.py

```diff
@@ -669,7 +675,7 @@ def sync(cls, repos, distro):
                         '--newest-only',
                         '--norepopath',
                         '-p',
-                        destination
+                        destination % repo_directories[repo]
```

Comment:
I don't think the ``% repo_directories[repo]`` piece is needed.  ``destination`` on its own already contains the right path (and no longer contains a %s to interpolate anyways)